src/org/jitsi/impl/neomedia/rtp/RTPEncodingDesc.java

```diff
@@ -63,6 +61,16 @@
      */
     private static final int FRAMES_HISTORY_SZ = 60;
 
+     /**
+      * The maximum time interval (in millis) an encoding can be considered
+      * active without new frames. This value corresponds to 7.5fps + 65 millis
+      * to compensate for network noise. If the network is clogged and we don't
+      * get a new frame within 200 millis, and if the encoding is being
+      * received, then we will ask for a new key frame (this is done in the
+      * JVB in SimulcastController).
+      */
+    private static final int SUSPENSION_THRESHOLD_MS = 200;
```

Comment:
this logic will apply to the screenshare stream as well, won't it?  will that framerate work there?

Reply:
I'll run some tests for that, but this code should not run for non-simulcast streams, and simulcast streams should have at least 7.5fps. If we have a simulcast stream that's lower than 7.5fsp may cause issues

Reply:
So yeah the code doesn't run in the screen sharing case but I tweaked the timeout in an attempt to avoid problems when Chrome activates simulcast for screen sharing.